src/vs/workbench/browser/actions/workspaceCommands.ts

```diff
@@ -235,7 +235,7 @@ CommandsRegistry.registerCommand({
 		return commandService.executeCommand('_files.newWindow', commandOptions);
 	},
 	description: {
-		description: 'Opens an new window depending on the newWindow argument.',
+		description: 'Opens an new window depending on the reuseWindow argument.',
```

Comment:
unrelated change sin this file?

Reply:
It is an accurate change to the description, but you can remove it if you like

Reply:
@jrieken removed

Reply:
In this case, it's ok. I've already reviewed all the changes but @jrieken brings up a really good point for future PRs.